source/mir/las/sum.d

```diff
@@ -1839,61 +1839,6 @@ nothrow unittest
     }
 }
 
-nothrow unittest
-{
-    import std.algorithm: map;
-    import std.array: array;
-    import std.meta: AliasSeq;
-
-    alias S = Summation;
-    alias sums = AliasSeq!(S.kahan, S.pairwise, S.naive, S.fast, S.precise);
-
-    enum double M = (cast(double)2) ^^ (double.max_exp - 1);
-    Tuple!(double[], double)[] tests = [
```

Comment:
I just proposed to include them because coverage was complaining - didn't know I was breaking so much :/